src/ic-websocket-cdk/src/types.rs

```diff
@@ -173,7 +173,7 @@ impl RegisteredGateway {
 
     /// Decrements the connected clients count by 1, returning the new value.
     pub(crate) fn decrement_clients_count(&mut self) -> u64 {
-        self.connected_clients_count -= 1;
```

Comment:
How can it go negative?

Reply:
It can't in theory, but being sure the canister doesn't trap for this is better